packages/web/src/components/shared/CancelSvg.js

```diff
@@ -3,20 +3,18 @@ import types from '@appbaseio/reactivecore/lib/utils/types';
 
 const CancelSvg = ({ onClick }) => (
 	<svg
+		onClick={onClick}
 		alt="Clear"
 		className="cancel-icon"
-		height="10"
-		onClick={onClick}
 		xmlns="http://www.w3.org/2000/svg"
-		viewBox="0 0 212.982 212.982"
+		height="20px"
+		viewBox="0 0 24 24"
+		width="20px"
+		fill="#000000"
```

Comment:
@mohdashraf010897 Hardcoded `fill` value could break for existing users. This is how we're merging classes. Make that change everywhere.
![alt image](https://i.imgur.com/K7cPmBt.png)

Reply:
Couldn't clearly understand? Does this work? have removed hard-coded fill from the svg file itself.

<img width="675" alt="Screenshot 2021-05-26 at 12 18 04 PM" src="https://user-images.githubusercontent.com/57627350/119614928-78347e80-be1c-11eb-9ce1-9f137e19da98.png">